Makefile

```diff
@@ -145,7 +145,6 @@ RUN_CMD?=go run
 
 .PHONY: run
 run:
-	@ls ./bundle/manifests/logging.openshift.io_*.yaml | xargs -n1 oc apply -f
```

Comment:
For running locally, this is creating a lot of confusion, as "make generate" does not modify the resources in the bundle directory. If are developing locally (with make run), the modified crd resources are in config/crd/bases, not in the bundle. I removed this because every time I do make run, my CR changes would get removed. The solution would be to use "oc apply -f config/crd/bases" to replace this line. WDYT?

Reply:
Do CRD bases include everything needed for the CRD?  What about permissions that also exist in the bundle?  AFAIK the CRDs are not the only requirement to run the operator which means even running locally you may not be able to access other parts due to permission problems

Reply:
Yeah, the CRD bases only include the crds itselves, thats right. But the manifests in the bundle do not change when make run/make generate happens, so this line always apply older resources into the cluster every time we do a make run. IMHO, this renders it useless for development.

Reply:
I have realized that we could just run "make bundle" which is as fast as make generate and will generate the correct resources in the bundle dir.